### ISSUE TITLE
Group files in 2.0

### ISSUE THREAD

**Issue by regs01** [windirstat/windirstat#86]
In WinDirStat 1 files were grouped into single expandable entry. But I can't find this option in 2.0. So directories with thousands of files take very long to expand. It would be good to have it at least as an option in 2.0 as well.

**Comment by NoMoreFood** [comment]
Correct, it's not in there right now.  Will consider re-adding to make it optional after an initial 2.0 release since it's quite a large change to make it work again with all the other changes we made.

**Comment by NoMoreFood** [comment]
I think we've improved the expansion performance well enough such that this is no longer necessary.  No plans to implement so closing.
